### ISSUE TITLE
Add room names to talk-related tasks

### ISSUE THREAD

**Issue by jrial** [jrial/fosdem-volunteers#64]
The Penta XML contains the room names, so it's trivial to add them to the tasks. That way volunteers know if they need to change rooms without consulting a separate schedule.

**Comment by emanuil-tolev** [comment]
We would likely need to support amendments if we do this. Do we just do 1 import at the moment or do we sync (e.g. every hour or day).

There is a potential problem introduced by automatic sync, with people signing up to moderate a talk they specifically want to see. If we change the talk that could be important to their decision on when to moderate. We could consider emailing the assigned people if the title in the slot they are assigned to is about to change. This we should deal with as a separate issue though, since we have this problem right now as well.

**Comment by jrial** [comment]
Yup, thought there was already an issue for that, but apparently it lived only in my head.

I'll open a separate issue before something happens to my head. ;)
